content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/saml-sso.md

```diff
@@ -43,7 +37,162 @@ We don’t synchronize user datastores, so make sure that all users:
+    - For these parameters, enter values from Azure AD:
+      - **IdP Issuer URL**: Azure AD Identifier
+      - **IdP Target URL**: Login URL
+      - **IdP Certificate**: Certificate (Base64)
+    - Save the **IdP Relay State** value that appears in red.
+1. In Azure AD, under **Basic SAML Configuration**, paste the **IdP Relay State** value into **Relay State**.
+1. [Test your configuration](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso#test-single-sign-on).
+
+### Duo
+
+To set up SAML SSO with Duo, read their [documentation](https://duo.com/docs/sso-generic).
+
+| For This Parameter in Cobalt | Enter This Value from Duo |
+|---|---|
+| IdP Issuer URL | Entity ID |
+| IdP Target URL | Single Sign-On URL |
+| IdP Certificate | Certificate |
+
+<br>
+
+| For This Parameter in Duo | Enter This Value from Cobalt |
+|---|---|
+| Entity ID | `https://api.cobalt.io/users/saml/metadata` |
+| Assertion Consumer Service (ACS) URL | `https://api.cobalt.io/users/saml/auth` |
+| Default Relay State | IdP Relay State |
+
+In Duo, complete the **SAML Response** section with:
+
+- **NameID format**: `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`
+- **NameID attribute**: `mail`
+- **Signature algorithm**: `SHA256`
+- **Signing options**: Select **Sign response** and **Sign assertion**.
+- **Map attributes**:
+  - **IdP Attribute**: enter `mail`.
+  - **SAML Response Attribute**: enter `email`.
+
+### Google
+
+For instructions on how to enable SAML SSO with Google, read their [guide](https://support.google.com/a/answer/6087519?hl=en).
+
+| For This Parameter in Cobalt | Enter This Value from Google |
+|---|---|
+| IdP Issuer URL | Entity ID |
+| IdP Target URL | SSO URL |
+| IdP Certificate | Certificate |
+
+<br>
+
+| For This Parameter in Google | Enter This Value from Cobalt |
+|---|---|
+| ACS URL | `https://api.cobalt.io/users/saml/auth` |
+| Entity ID | `https://api.cobalt.io/users/saml/metadata` |
+| Start URL | IdP Relay State |
+
+In the Google Admin console, configure the following:
+
+<!--- On the **Service Provider Details** page, leave the **Signed Response** option unselected (default).-->
+- On the **Attribute Mapping** page, add an attribute `email`, and select **Basic Information** and **Primary Email**.
+
+Once you've completed the setup, your application for Cobalt appears in the Google Workspace.
+
+### Okta
+
+To learn how to configure Okta as IdP, read their [documentation](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SAML.htm?cshid=ext_Apps_App_Integration_Wizard-saml). We recommend that you create a SAML integration for Cobalt manually.
+
+You can also use the preset configuration. Okta has an [integration with Cobalt](https://www.okta.com/integrations/cobalt/) for SAML. To learn more, see [how to configure SAML SSO with Okta](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Cobalt.html).
```

Comment:
Replace "We recommend that you create a SAML integration for Cobalt manually.

You can also use the preset configuration." with "We recommend creating a SAML integration for Cobalt manually. You can also use the Cobalt app (https://www.okta.com/integrations/cobalt/) on Okta.